app/src/pages/view_map.py

```diff
@@ -28,64 +36,112 @@
 assetsDir = os.path.abspath(os.path.join(srcDir, "assets"))  # go up one level to src
 load_dotenv()
 
-# global variables
-map_layer_dict = {
-    "Subbasins": "https://duwamish-pilot.s3.amazonaws.com/stac/Subbasin.geojson",
-    "Reaches": "https://duwamish-pilot.s3.amazonaws.com/stac/Reach.geojson",
-    "Junctions": "https://duwamish-pilot.s3.amazonaws.com/stac/Junction.geojson",
-    "Reservoirs": "https://duwamish-pilot.s3.amazonaws.com/stac/Reservoir.geojson",
-}
-
 
 def view_map():
     if "session_id" not in st.session_state:
         init_session_state()
 
     st.session_state.log_level = LOG_LEVEL
 
-    if st.session_state["init_map_data"] is False:
+    st.sidebar.markdown("## Toolbar")
+    st.session_state["pilot"] = st.sidebar.selectbox(
+        "Select a Pilot Study",
+        [
+            "Trinity",
+        ],
+        index=0,
+    )
+
+    if st.session_state["init_pilot"] is False:
         with st.spinner("Initializing datasets..."):
-            init_map_data(map_layer_dict)
-            st.session_state["init_map_data"] = True
-            st.success("Complete! Map data is now ready for exploration.")
+            init_pilot(st.session_state["pilot"])
+            st.session_state["init_pilot"] = True
+            st.success("Complete! Pilot data is now ready for exploration.")
 
-    st.sidebar.markdown("## Toolbar")
     st.session_state["basemap"] = st.sidebar.selectbox(
-        "Select Basemap",
+        "Select a Basemap",
         ["OpenStreetMap", "ESRI Satellite", "Google Satellite"],
         index=0,
     )
-    st.session_state["gage_param"] = st.sidebar.selectbox(
-        "Select Gage Parameter",
-        ["Streamflow", "Stage", "Precipitation"],
+
+    st.session_state["gage_plot"] = st.sidebar.selectbox(
```

Comment:
Why don't we just display all of the gage plots when a gage is selected?

Reply:
Also, could we set the size of the plots on the page? They appear really large and blurry.